ricecooker/utils/videos.py

```diff
@@ -48,13 +49,24 @@ def extract_thumbnail_from_video(fpath_in, fpath_out, overwrite=False):
         midpoint = float(re.search("\d+\.\d+", str(result)).group()) / 2
         # scale parameters are from https://trac.ffmpeg.org/wiki/Scaling
         scale = "scale=400:225:force_original_aspect_ratio=decrease,pad=400:225:(ow-iw)/2:(oh-ih)/2"
-        command = ['ffmpeg',"-y" if overwrite else "-n", '-i', str(fpath_in), "-vf", scale, "-vcodec", "png", "-nostats",
-                  '-ss', str(midpoint), '-vframes', '1', '-q:v', '2', "-loglevel", "panic", str(fpath_out)]
+        command = ['ffmpeg', "-y" if overwrite else "-n", '-i', str(fpath_in), "-vf", scale, "-vcodec", "png",
+                   "-nostats",
+                   '-ss', str(midpoint), '-vframes', '1', '-q:v', '2', "-loglevel", "panic", str(fpath_out)]
         subprocess.check_output(command, stderr=subprocess.STDOUT)
     except subprocess.CalledProcessError as e:
         raise ThumbnailGenerationError("{}: {}".format(e, e.output))
 
 
+def extract_duration_of_video(fpath_in):
+    try:
+        result = subprocess.check_output(['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of',
```

Comment:
Would this same command work for an audio file too? In which case, we can make the function name more generic like `extract_duration_of_media`.

Reply:
Yes it can be change.  This work for both audio and video. I tested with both.

Reply:
That would be good - can also add the duration pieces you've added to video file classes to audio files too.

Reply:
Still need to update the method name here.

Reply:
Name changed